README.md

```diff
@@ -21,19 +21,19 @@ The Node.js project uses an [open governance model](./GOVERNANCE.md). The
 # Table of contents
 
 * [Support](#support)
-* [Release Types](#release-types)
+* [Release types](#release-types)
   * [Download](#download)
     * [Current and LTS Releases](#current-and-lts-releases)
```

Comment:
It's probably worth fixing this one as well.
```suggestion
    * [Current and LTS releases](#current-and-lts-releases)
```

Reply:
change applied!